common/types/activation.go

```diff
@@ -223,9 +200,17 @@ func (challenge *NIPostChallenge) String() string {
 // structure is serialized and signed. It includes the header fields, as well as the larger fields that are only used
 // for validation: the NIPost and the initial Post.
 type InnerActivationTx struct {
```

Comment:
do you have a test that guarantee that InnerActivationTx has everything public field of ActivationTxHeader?
i.e. how do we keep these two struct in sync over time?